app/src/main/java/com/teamltt/carcare/activity/HomeActivity.java

```diff
@@ -193,4 +188,36 @@ public void toggleLogging(MenuItem item) {
         }
 
     }
+
+    public void displayStaticData() {
+        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(this);
+        String syncConnPref = preferences.getString(SettingsActivity.KEY_PREF_SYNC_CONN, "");
+
+        TextView tvStaticDataName = ((TextView) findViewById(R.id.tvStaticDataName));
+        TextView tvStaticDataValue = ((TextView) findViewById(R.id.tvStaticDataValue));
+
+        String tvTextName = "";
+        String tvTextValue = "";
+
+        if (preferences.getBoolean("sEngineTemp", false)) {
```

Comment:
This String should either be a constant in some class (perhaps SettingsActivity) or placed in the strings.xml resource.